arma3/@task_force_radio/addons/task_force_radio/functions/fn_ProcessTangent.sqf

```diff
@@ -1,27 +1,27 @@
 /*
```

Comment:
I'd prefer you not changing files just because of formatting. But it doesn't matter that much because we will reformat everything on 1.0 anyway.